### PR TITLE
Add two files in Fundamentals

### LINKED ISSUE #1537: [Idea]: Introduce a new subsection in the Fundamental
### Idea Title

Add Loops and Conditional Statements to the Fundamental Section

### Idea Description

Introduce a new subsection in the Fundamentals section covering loops (for, while, do while) and conditional statements (if-else, switch). Provide explanations and example problems for users to practice basic programming constructs.

### Potential Benefits

- [X] Improves user experience
- [X] Enhances learning
- [ ] Adds a new feature
- [ ] Improves code quality
- [ ] Other

### Implementation Suggestions (Optional)

_No response_

### PR DESCRIPTION
**Pull request by AnushkaChouhan25**
### Description
Add Loops and Conditional Statements to the Fundamental Section

Fixes #1537

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/fundamentals/Conditional Statement.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+id: Coditional Statement
```
Comment (ajay-dhangar) [inline docs/fundamentals/Conditional Statement.md]: replace `id: Coditional Statement` with `id: coditional-statement`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Update your code based on suggestions

**Comment by AnushkaChouhan25** [comment]
@ajay-dhangar Done

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**